pkg/scheduler/factory/factory.go

```diff
@@ -319,31 +284,18 @@ func NewConfigFactory(args *ConfigFactoryArgs) Configurator {
 	return c
 }
 
-func (c *configFactory) GetHardPodAffinitySymmetricWeight() int32 {
+// GetHardPodAffinitySymmetricWeight is exposed for testing.
+func (c *Configurator) GetHardPodAffinitySymmetricWeight() int32 {
 	return c.hardPodAffinitySymmetricWeight
 }
 
-func (c *configFactory) GetSchedulerName() string {
```

Comment:
Review note for me: Since fbf3d2b84cae205319fbbac5115bc7d8e1df1f4e this function has been unused and we can remove schedulerName also as an internal variable.